src/lib/frontend/models.mli

```diff
@@ -30,16 +30,17 @@
 
 (** {1 Models module} *)
 
+type t = {
+  propositional : Expr.Set.t;
+  constants : ModelMap.t;
+  functions : ModelMap.t;
+  arrays : ModelMap.t;
+}
+
 (** Print the given counterexample on the given formatter with the
     corresponding format set with Options.get_output_format.
     - functions: the functions of the model;
     - constants: the variables of the model;
     - arrays: (experimental) the arrays of the model.
 *)
```

Comment:
We have to up to date this comment.

Reply:
Sure. Do you want to do it in this PR, or create an issue to do it later?

Reply:
I did it in this PR.